workspace/__lib__/xod/math/sin/patch.test.tsv

```diff
@@ -0,0 +1,11 @@
+RAD	OUT
+0	0
+1	0.841471~
+-1	-0.841471~
+180	-0.801153~
+270	-0.176046~
+360	0.958916~
+-520	0.997795~
```

Comment:
What’s the point to test against 180, 270, 360 _radians_? I would expect values like π/2, π, 3π/2, 2π here